pages.es/linux/mopac.md

```diff
@@ -0,0 +1,12 @@
+# mopac
+
+> MOPAC (Molecular Orbital PACkage) es un programa semiempírico de química cuántica basado en la aproximación NDDO de Dewar y Thiel.
+> Más información: <https://github.com/openmopac/mopac>.
+
+- Realiza los cálculos a partir de un archivo de entrada (`.mop`, `.dat`, y `.arc`):
+
+`mopac {{ruta/al/archivo_de_entrada}}`
+
+- Mínimo ejemplo de trabajo con HF que escribe en el directorio actual y transmite el archivo de salida:
```

Comment:
The command example description does not seem to be in imperative. I do not know to re-write it, any ideas or thoughts @kant?

Reply:
It is perfect for me